README.md

```diff
@@ -5,7 +5,7 @@ Google load balancer project.
 
 | Environment Variable | Function |
 | ---------- | ------- |
-| SSL_POLICY_NAME  | Given a name will assert a Policy with that name exists. Hardcoded to minimum TLS 1.2 and MODERN profile |
+| SSL_POLICY_NAME  | Given a name will assert a Policy with that name exists. Hardcoded to minimum TLS 1.2 and RESTRICTED profile |
```

Comment:
The documentation should reflect the new environment variables.

Reply:
Fixed by https://github.com/commercetools/gcp-ssl-policy-asserter/pull/3/commits/6f590c87e8855b8392419a2bb64a92a384f2a23c